app_dart/test/request_handlers/push_gold_status_to_github_test.dart

```diff
@@ -552,63 +645,6 @@ void main() {
           ));
         });
       });
-
-      test(
-          'Completed pull request does not skip follow-up prs with early return',
```

Comment:
To verify, this was removed because this was the performance issue?

Reply:
That was an error. 😅  It's back.